remindme-api/src/main/java/it/andreafailli/remindme/RemindMeApiApplication.java

```diff
@@ -31,20 +32,9 @@
 public class RemindMeApiApplication {
 
 	public static void main(String[] args) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)